### DISCUSSION TITLE
pytorch version

### DISCUSSION THREAD

**Discussion by shoehangxh** [Lightning-AI/pytorch-lightning#10386] [General]
some of the old programs need the enviroment of pytroch 1.4, but the pytorch-ligntning needs a pytorch more than 1.6
so is there some methods to make the pytorch-ligntning feasible on the pytorch 1.4?

**Comment by akihironitta** [comment]
@shoehangxh Hi, you can still give it a try with `torch==1.4` (but no support from our side). See also #9543.
